test/Microsoft.ML.Tests/TrainerEstimators/LbfgsTests.cs

```diff
@@ -80,14 +81,24 @@ public void TestLogisticRegressionStats_MKL()
                 s.StdComputer = new ComputeLRTrainingStdThroughHal();
             }));
 
-            var transformerChain = pipe.Fit(dataView) as TransformerChain<BinaryPredictionTransformer<ParameterMixingCalibratedPredictor>>;
+            var transformer = pipe.Fit(dataView) as TransformerChain<BinaryPredictionTransformer<ParameterMixingCalibratedPredictor>>;
 
-            var linearModel = transformerChain.LastTransformer.Model.SubPredictor as LinearBinaryModelParameters;
+            var linearModel = transformer.LastTransformer.Model.SubPredictor as LinearBinaryModelParameters;
             var stats = linearModel.Statistics;
             LinearModelStatistics.TryGetBiasStatistics(stats, 2, out float stdError, out float zScore, out float pValue);
```

Comment:
>LinearModelStatistics.TryGetBiasStatistics(stats, 2, out float stdError, out float zScore, out float pValue); [](start = 12, length = 109)

This is the first thing that does not make much sense to me. It seems like `linearModel.Statistics` should offer similar accessors through an instance method. (Why `out` parameters? Why not just properties on the original structure?) This is just really complicated.

Reply:
+1

As Tom pointed out, the API for getting statistics needs some cleanup.  linearModel.Statistics or some other simple 1-liner should be the only code a user needs.

---
In reply to: [244374354](https://github.com/dotnet/machinelearning/pull/1979#discussion_r244374354) [](ancestors = 244374354)